core/src/test/java/kafka/test/junit/ZkClusterInvocationContext.java

```diff
@@ -206,6 +206,22 @@ public void startBroker(int brokerId) {
             findBrokerOrThrow(brokerId).startup();
         }
 
+        @Override
```

Comment:
I guess that we don't need those extra waiting. Or could you please loop the tests on your local to check the race condition?

Reply:
> I guess that we don't need those extra waiting. Or could you please loop the tests on your local to check the race condition?

Hi @chia7712 ,
As discussion offline, this is normal flow to deal `ZK` mode so we have to keep it.